[2025-08-04T14:43:36.861+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=load_random_users_to_rds/run_id=manual__2025-08-04T14:33:06.802009+00:00/task_id=load_users_to_rds permission to 509
[2025-08-04T14:43:37.034+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=load_random_users_to_rds/run_id=manual__2025-08-04T14:33:06.802009+00:00/task_id=load_users_to_rds permission to 509
[2025-08-04T14:43:37.135+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_random_users_to_rds.load_users_to_rds manual__2025-08-04T14:33:06.802009+00:00 [queued]>
[2025-08-04T14:43:37.160+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_random_users_to_rds.load_users_to_rds manual__2025-08-04T14:33:06.802009+00:00 [queued]>
[2025-08-04T14:43:37.161+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 4
[2025-08-04T14:43:37.224+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_users_to_rds> on 2025-08-04 14:33:06.802009+00:00
[2025-08-04T14:43:37.244+0000] {standard_task_runner.py:60} INFO - Started process 567 to run task
[2025-08-04T14:43:37.254+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'load_random_users_to_rds', 'load_users_to_rds', 'manual__2025-08-04T14:33:06.802009+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/user.py', '--cfg-path', '/tmp/tmpoymz9y2s']
[2025-08-04T14:43:37.261+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask load_users_to_rds
[2025-08-04T14:43:37.400+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=load_random_users_to_rds/run_id=manual__2025-08-04T14:33:06.802009+00:00/task_id=load_users_to_rds permission to 509
[2025-08-04T14:43:37.424+0000] {task_command.py:423} INFO - Running <TaskInstance: load_random_users_to_rds.load_users_to_rds manual__2025-08-04T14:33:06.802009+00:00 [running]> on host 9edf237adf6e
[2025-08-04T14:43:38.192+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_random_users_to_rds' AIRFLOW_CTX_TASK_ID='load_users_to_rds' AIRFLOW_CTX_EXECUTION_DATE='2025-08-04T14:33:06.802009+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-04T14:33:06.802009+00:00'
[2025-08-04T14:46:30.866+0000] {logging_mixin.py:188} INFO - 500 users loaded successfully.
[2025-08-04T14:46:31.434+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-04T14:46:32.008+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=load_random_users_to_rds, task_id=load_users_to_rds, execution_date=20250804T143306, start_date=20250804T144337, end_date=20250804T144632
[2025-08-04T14:46:33.279+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-04T14:46:34.057+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
